### DISCUSSION TITLE
greetings & STT audio file?

### DISCUSSION THREAD

**Discussion by i2rllc** [coqui-ai/STT#2331] [Q&A] [answered]
Hello, Thank you for having me. I am new to python and coqui and STT but am looking to get some experience using coqui to deploy on a project I am trying. I already ran the model manager and it looks very cool. Is there a way to feed an audio file to coqui for STT? If so, can you please share instructions? Please be gentle, I am new to all of this. Thank you!

**Comment by wasertech** [comment, marked as answer]
> Is there a way to feed an audio file to coqui for STT?

Sure the basic command is:
```bash
stt --model output_graph.tflite --scorer kenlm.scorer  --audio audio.wav
```
You audio file must be in WAV format 16 bit signed interger with 16K Hz sample rate.
